_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -158,6 +158,10 @@ public function get_item_schema() {
 				),
 				'children' => array(
 					'items' => array(
+						'count'  => array(
```

Comment:
Would it be worth marking this explicitly as `required => false` just so it's ultra clear the property may not be present?

We need to be careful we don't invalidate the Calypso schema just because it's expecting a count.

Reply:
Hm, that doesn't feel right. Maybe we could prefix the description with "Optional."?

Reply:
Not a blocker.